bioio_ome_zarr/writers/ome_zarr_writer_v3.py

```diff
@@ -0,0 +1,497 @@
+import sys
+
+if sys.version_info < (3, 11):
```

Comment:
this really isn't needed anymore

Reply:
Wait what why is this here then?

Reply:
resolved in [2b11e82](https://github.com/bioio-devs/bioio-ome-zarr/pull/41/commits/2b11e82b50013e27943f10a1c66324b3bd096271)